modules/exploits/windows/http/sharepoint_ssi_viewstate.rb

```diff
@@ -90,6 +90,7 @@ def initialize(info = {})
     register_options([
       OptString.new('TARGETURI', [true, 'Base path', '/']),
       OptString.new('VALIDATION_KEY', [false, 'ViewState validation key']),
+      OptString.new('Cookie', [false, 'SharePoint cookie if you have one']),
```

Comment:
Oh, actually capitalize this option, since it's a normal option. The advanced options are CamelCase. Not a rule, just a guideline.

```suggestion
      OptString.new('COOKIE', [false, 'SharePoint cookie if you have one']),
```